static/app/views/performance/browser/webVitals/components/performanceBadge.tsx

```diff
@@ -0,0 +1,39 @@
+import {useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+
+type Props = {
+  status: 'good' | 'meh' | 'poor';
+};
+
+export const scoreToStatus = (score: number) => {
+  if (score >= 90) {
+    return 'good';
+  }
+  if (score >= 50) {
+    return 'meh';
+  }
+  return 'poor';
+};
+
+export function PerformanceBadge({status}: Props) {
+  const theme = useTheme();
+  const text = status === 'good' ? 'GOOD' : status === 'meh' ? 'MEH' : 'POOR';
```

Comment:
For readability's sake, we should use `toUpperCase()` here instead

Reply:
same as above!